emData/download.sh

```diff
@@ -209,14 +220,27 @@ then
   rm -f LUTs.tar.gz
 fi
 
-# Run scripts to generate top functions in TrackletAlgorithm/
-./generate_IR.py
+# Run scripts to generate top functions in TopFunctions/
+mkdir -p ../TopFunctions/tmp
+mv ../TopFunctions/*.{h,cc} ../TopFunctions/tmp/
+mkdir -p ../TopFunctions/ReducedConfig
+./generate_IR.py LUTsReduced/wires.dat
+./generate_VMR.py -a -w LUTsReduced/wires.dat
+./generate_TC.py LUTsReduced/wires.dat
+./generate_PR.py LUTsReduced/wires.dat
+./generate_MC.py LUTsReduced/wires.dat
+./generate_TB.py LUTsReduced/wires.dat
+./generate_ME.py -s -w LUTsReduced/wires.dat
+mv ../TopFunctions/*.{h,cc} ../TopFunctions/ReducedConfig/
```

Comment:
Seems like rather than moving existing code to a temporary directory so that you can issue a general `mv` command, each of the generate scripts should have an option to output the files to a specific directory (i.e. https://github.com/cms-L1TK/firmware-hls/blob/master/emData/generate_ME.py#L183). Then you could avoid these extra lines of code here.